tests/vic-machine.bats

```diff
@@ -123,6 +124,12 @@ teardown() {
     run "$docker" "${docker_cmd[@]}" start "$name"
     assert_success
 
+    run wget "$vicadmin_ip"/container-logs.tar.gz --no-check-certificate -O install-container-logs.tar.gz
```

Comment:
Realizing you'll probably need to capture logs in the teardown() instead.  Otherwise if a prior assertion fails, we won't reach this point and no logs are captured for debugging.